audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
```

Comment:
`camelcase` won't flag non-camelcase variables accessed for comparison, so you don't need to turn off the rule here